### PR TITLE
Added new doc - subnet participants

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
- Added new document - Subnet participants

### CODE REVIEW

File: docs/feature-guides/subnet-participants.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+- Own user experience for people who want to use a Subnet via a wallet
+- Deposit of STX from Stacks main chain to a Subnet
+- Connect to Subnets for mints and transfers
+- Support withdrawals of STX and NFTs from Subnet to Stacks main chain
+- Display Subnet address balances
+
+### End Users
+
+- Users interact with Subnets to experience faster throughput and lower latency for specific use cases (dEFI trades or purchasing or trading NFTs).
```
Comment (jbencin) [inline docs/feature-guides/subnet-participants.md:67]:
```suggestion
- Users interact with Subnets to experience faster throughput and lower latency for specific use cases (DeFi trades or purchasing or trading NFTs).
```

File: docs/feature-guides/subnet-participants.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
```
Comment (jbencin) [inline docs/feature-guides/subnet-participants.md:68]: Extra newline here:

```suggestion
```

**Review by jbencin — COMMENTED** [review]
I left some grammar/punctuation suggestions. Also, I didn't mark these but some notes on consistency:
- We need to clarify whether it's "subnets" or "Subnets", and update all docs accordingly
- I'm not sure if bullet points should end with a "." or not (I don't think so, I don't do it), but use the same convention throughout the document

**Comment by LakshmiLavanyaKasturi** [comment]
Fixed issues. @jbencin Please review. Thanks!

**Review by saralab — APPROVED** [review]

File: docs/feature-guides/subnet-participants.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+### Supporting UseCase (Example: NFT Collection Mint)
+
+- A supporting use case refers to the event that drives people to use a subnet, for example, a mint for a popular NFT collection.
+- The supporting use case includes any other parties involved with the use case. In the example above, it could be the artist for a mint collection. The supporting application may be independent of miners.
+
+**Key Activities**
+
+Must agree to subnet parameters and incentives—for example, transaction fees, duration, and performance requirements.
+
+## Secondary participants
```
Comment (LakshmiLavanyaKasturi) [inline docs/feature-guides/subnet-participants.md:51]:
```suggestion
## Secondary Participants
```

**Pull request merged**